### PR TITLE
Update Grunt Rewards

### PR DESCRIPTION
**Pull request by TurtIeSocks**
- Updated Grunt Rewards
- Added Third Encounter
- Formatting

### CODE REVIEW

**Comment by Mygod** [comment]
https://github.com/TurtIeSocks/MapJS/pull/18

**Review by versx — APPROVED** [review]

**Review by Mygod — APPROVED** [review]
Pokestop popup is too long but I can live with it for now.

**Review by clburlison — CHANGES_REQUESTED** [review]
The pop is now so big it auto closes on smaller phones. The iPhone 7 has a 4.7" screen and can no longer see this information.

**Comment by TurtIeSocks** [comment]
image
image
image

@clburlison @Mygod better?

**Comment by TurtIeSocks** [comment]
image

Before & After

**Review by versx — APPROVED** [review]

**Comment by versx** [comment]
> image
> 
> Before & After

Might be able to reduce it another 10 pixels or so if you delete the double line breaks between `Last Updated` text and the navigation buttons.

**Comment by clburlison** [comment]
I'm no longer near my laptop so can't test but like the screenshots changes.

**Review by clburlison — APPROVED** [review]
Approving but can't validate the latest changes on a test device.

File: static/js/index.js [hunk 1]
```diff
@@ -3071,23 +3071,21 @@ function getPokestopPopupContent (pokestop) {
...
+            content += '<b>Reward:</b> ' + getQuestReward(reward) + '<br>';
         });
 
         content += '<br>';
     }
 
     const updatedDate = new Date(pokestop.updated * 1000);
     if (updatedDate) {
-        content += '<small><b>Last Updated:</b> ' + updatedDate.toLocaleDateString() + ' ' + updatedDate.toLocaleTimeString() + ' (' + getTimeSince(updatedDate) + ')<br></small>';
+        content += '<center><small><b>Last Updated:</b> ' + updatedDate.toLocaleDateString() + ' ' + updatedDate.toLocaleTimeString() + ' (' + getTimeSince(updatedDate) + ')<br></small></center>';
```
Comment (Mygod) [inline static/js/index.js:3085]: Avoid using center/small. Use CSS.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

**Comment by TurtIeSocks** [comment]
image
image
image

Alright, well before I get too off course with this pr, I cleaned up a few things for quests/invasions. I'll do more cleanup in a later PR. Let me know if everything looks okay and I'll go ahead and merge.

File: static/css/index.css [hunk 2]
```diff
@@ -306,7 +306,37 @@ fieldset[disabled] .btn-size.active {
...
 }
+.table-invasion {
+    margin-left: auto;
+    margin-right: auto;
+    font-weight: bold;
+}
+.table-invasion td:first-child {
+    padding-right: 15px;
+}
+.table-invasion td:nth-child(2n) {
```
Comment (Mygod) [inline static/css/index.css:333]: `td:nth-child(2)`

File: static/css/index.css [hunk 3]
```diff
@@ -306,7 +306,37 @@ fieldset[disabled] .btn-size.active {
...
+    width: 100px
+}
+.table-invasion td:last-child {
+    padding-left: 13px;
```
Comment (Mygod) [inline static/css/index.css:337]: 13 is an odd (prime) number... I think 12px will work better on screens with e.g. 150% scale.

File: static/css/index.css [hunk 4]
```diff
@@ -325,7 +355,7 @@ img {
     image-rendering: crisp-edges;
     image-rendering: -webkit-optimize-contrast;
 }
-
+
```
Comment (Mygod) [inline static/css/index.css:358]: uh

**Pull request merged**